app/src/main/java/com/onewelcome/showcaseapp/ui/screens/HomeScreen.kt

```diff
@@ -64,12 +64,27 @@ private fun Section(section: SectionItem, onNavigateToSection: (route: String) -
   Card(
     modifier = Modifier
       .fillMaxWidth()
-      .padding(16.dp),
+      .padding(Dimensions.standardPadding),
     onClick = {
       onNavigateToSection.invoke(section.navigation.route)
     }
   ) {
-    Text(modifier = Modifier.padding(16.dp), text = section.title)
+    Row(
+      modifier = Modifier.padding(Dimensions.standardPadding),
+      verticalAlignment = Alignment.CenterVertically,
+      horizontalArrangement = Arrangement.spacedBy(Dimensions.standardPadding)
+    ) {
+      Text(
+        modifier = Modifier
```

Comment:
Same here, can be removed if `Row` will have `.fillMaxWidth()` modifier.

Reply:
In case the text is longer than one line, then the arrow will disappear with the above approach :(